test/unittest.js

```diff
@@ -2,8 +2,16 @@
 require('source-map-support').install();
 
 var apiCompatibilityChecks = require('./api_compatibility.js');
+var noopImplementationTests = require('./noop_implementation.js');
+var opentracingAPITests = require('./opentracing_api.js');
+var opentracing = require('../debug.js');
```

Comment:
I think it makes sense to run both the debug and non-debug versions of noop tracer through the harness